.gitignore

```
@@ -15,3 +16,6 @@ plugins.txt
 */.project
 */.settings
 
+# Ignore DS_Store (Mac)
+**/.DS_Store
```

.DS_Store on its own should be fine
https://github.com/github/gitignore/blob/320a610b9ffba3ff28166370b6dd9f73096e34b7/Global/macOS.gitignore#L2

```suggestion
.DS_Store
```

---

Suggestion applied 🙌 Nice new feature

---

Suggestion applied 🙌 Nice new feature 😎